content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable. Please be aware
+    that cluster names may be autogenerated by your Kubernetes cloud provider.
+    Therefore, the cluster name you specify during creation of your cluster may
+    be different than the name of the cluster expected here. This will be evident
+    by invalid x509 certificate errors in the Istio logs after completing
+    installation.
+
+    {{< text bash >}}
+    $ CLUSTER_NAME="<the name of your cluster>"
+    {{< /text >}}
+
+1. Create a working directory for files generated in this documentation.
+ 
+    {{< text bash >}}
+    $ mkdir -p "${HOME}"/"${CLUSTER_NAME}"
+    {{< /text >}}
+
+1. Execute the following commands on each cluster in the mesh where a virtual
```

Comment:
+1 - this is an example, demo, etc - NOT A PRODUCTION SETUP.

Users should install istio for production using the recommended process. If they happened to use an external CA - as one of the install methods recommend - they should integrate with that. If they used Istio CA - they should use it.

Reply:
SGTM

Reply:
Which install method recommends using an external CA? Can you link the document please? I don'tsee anything in the install tree that explains this. There is content in the concepts documents that explain this, although concepts are not install instructions.

Reply:
I will add a callout as per the request in https://github.com/istio/istio.io/pull/7248#discussion_r424079771 and also indicate that the certificates, while secure, are not for production use as per comment https://github.com/istio/istio.io/pull/7248#discussion_r424091418. Suitable?

Reply:
Our install guides are a bit of a mix of production and demo unfortunately. We're trying to be as opinionated as we can but still be vendor agnostic. So that means we can mention ILB and external CA but the actual instructions  here won't be able to leverage them.

Other guides can discuss the details of integration on a particular cloud provider or with a specific external CA, but that information wouldn't be under install.

With all of that said, @sdake is basically doing the same cert provisioning that I am in the [multicluster docs](https://docs.google.com/document/d/1wzoFqLHw-dEoKPVi1w5tlckWTn78qpYNbC5fEwp-Mw4/edit#heading=h.vu9te4khxjut), so the two docs will be consistent and I believe correct for the purposes of an installation guide.

@sdake @costinm @howardjohn WDYT?

@myidpt @rcaballeromx do we have docs for setting up an external CA that we can link to here (and from the multicluster docs)?

Reply:
From @rcaballeromx:

> @nmittler The doc with the instructions to plug in an existing CA are here: https://istio.io/docs/tasks/security/plugin-ca-cert/

@costinm is this what you were referring to when you mentioned [using an external CA](https://github.com/istio/istio.io/pull/7248/files/350dfa37982e24e090ee5be10e15f64e3944a22c#r424091418)? It sould seem that @sdake is already doing that in this PR.

Reply:
@nmittler thanks for the review. I have [addresssed](https://github.com/istio/istio.io/pull/7248/commits/eddd5cf9cde34b53317db1ae3ee728abbf31c465) @costinm's and @howardjohn's objection